installers/gcp-marketplace/application.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: app.k8s.io/v1beta1
+kind: Application
+metadata:
+  name: '${OPERATOR_NAME}'
+  labels:
+    app.kubernetes.io/name: '${OPERATOR_NAME}'
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: '${OPERATOR_NAME}'
+  componentKinds:
+    - { group: core, kind: ConfigMap }
+    - { group: core, kind: Secret }
+    - { group: core, kind: Service }
+    - { group: apps, kind: Deployment }
+    - { group: batch, kind: Job }
+  descriptor:
```

Comment:
This information appears in GCP Console.

Reply:
That's fine.